### PR TITLE
Fixed bug that was preventing popover values from saving.

### PR DESCRIPTION
**Pull request by AWSHurneyt**
*Issue #, if available:*

*Description of changes:*
1. Implemented default values logic for visual graph aggregation triggers to prevent popover fields from saving as undefined.
2. Removed unnecessary validation from subsequent calls to those values
3. Fixed syntax errors in `switch` statements.

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

### CODE REVIEW

File: public/pages/CreateMonitor/containers/DefineMonitor/DefineMonitor.js [hunk 1]
```diff
@@ -204,7 +204,8 @@ class DefineMonitor extends Component {
         _.set(monitor, 'triggers', []);
 
         switch (searchType) {
-          case SEARCH_TYPE.QUERY || SEARCH_TYPE.GRAPH:
+          case SEARCH_TYPE.QUERY:
```
Comment (leeyun-amzn) [inline public/pages/CreateMonitor/containers/DefineMonitor/DefineMonitor.js:207]: Should there be something defined here?

Comment (AWSHurneyt) [reply public/pages/CreateMonitor/containers/DefineMonitor/DefineMonitor.js:207]: My understanding is that [javascript's `switch` syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch) doesn't use `||` to indicate `or`. You instead list the cases one after the other, followed by the action, to indicate that the function should perform that action in all of those cases. So in this context, the function will execute line 209 in cases where `searchType` is `GRAPH` or `QUERY`.

File: public/pages/CreateTrigger/containers/CreateTrigger/CreateTrigger.js [hunk 2]
```diff
@@ -152,7 +152,8 @@ export default class CreateTrigger extends Component {
     _.set(monitorToExecute, 'triggers', triggers);
 
     switch (searchType) {
-      case SEARCH_TYPE.QUERY || SEARCH_TYPE.GRAPH:
+      case SEARCH_TYPE.QUERY:
```
[the remark this answers is not in the document]
Comment (AWSHurneyt) [reply public/pages/CreateTrigger/containers/CreateTrigger/CreateTrigger.js:155]: Replied above as well, but I'll copy/paste the response here as well.

My understanding is that [javascript's `switch` syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch) doesn't use `||` to indicate `or`. You instead list the cases one after the other, followed by the action, to indicate that the function should perform that action in all of those cases. So in this context, the function will execute line 209 in cases where `searchType` is `GRAPH` or `QUERY`.

[the remark this answers is not in the document]
Comment (leeyun-amzn) [reply public/pages/CreateTrigger/containers/CreateTrigger/CreateTrigger.js:155]: Got it 👍

**Review by leeyun-amzn — APPROVED** [review]

**Pull request merged**